ios/Plugin/Plugin.swift

```diff
@@ -1,17 +1,46 @@
 import Foundation
 import Capacitor
+import Sentry
 
-/**
- * Please read the Capacitor iOS Plugin Development Guide
- * here: https://capacitorjs.com/docs/plugins/ios
- */
 @objc(SentryCapacitor)
 public class SentryCapacitor: CAPPlugin {
-
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
-        call.success([
-            "value": value
-        ])
+    
+    @objc public func startWithOptions(capOptions: CAPPluginCall) {
+        SentrySDK.start { options in
+            options.dsn = capOptions.getString("dsn") ?? ""
+            options.debug = capOptions.getBool("debug") ?? false
+            options.environment = capOptions.getString("environment") ?? "production"
+            options.enableAutoSessionTracking = capOptions.getBool("enableAutoSessionTracking") ?? false
```

Comment:
The default value for this field is `true` but to avoid duplicating it here I suggest checking whether a value existed in the map and only then mutating the `options` object. This way we're free to change the default values in the iOS SDK without having to remember/change here too.